packages/fxa-settings/src/components/LinkRememberPassword/index.test.tsx

```diff
@@ -35,19 +35,21 @@ describe('LinkRememberPassword', () => {
 
     expect(rememberPasswordLink).toHaveAttribute(
       'href',
-      '/signin?email=bleep%40bloop.com'
+      '/signin?email=johndope%40example.com'
```

Comment:
not gonna lie I miss bleepbloop

Reply:
I vote we replace `johndope` with `bleepbloop`